lib/postprocess/image.js

```diff
@@ -91,15 +91,19 @@ module.exports = async files => {
           'base64'
         )
       } else {
-        const response = await limit(() => axios({
-          url: imageConfig.src,
-          responseType: 'arraybuffer',
-          method: 'GET'
-        }))
-        const contentType =
-          response.headers['content-type'] || response.headers['Content-Type']
-        imageExtensions = contentType.split('/')[1]
-        imageBuffer = Buffer.from(response.data)
+        try {
+          const response = await limit(() => axios({
+            url: imageConfig.src,
+            responseType: 'arraybuffer',
+            method: 'GET'
+          }))
+          const contentType =
+            response.headers['content-type'] || response.headers['Content-Type']
+          imageExtensions = contentType.split('/')[1]
+          imageBuffer = Buffer.from(response.data)
+        } catch (err) {
+          console.warn('Failed to read image', { imageConfig }, { err });

```

Comment:
question: where will this `console.warn` message be logged?

Reply:
@michelfernandes Since this is running inside a container, and logging to `stdout`, this will be captured by the logs of the container and shipped per the container's log configuration. [This](https://eu-central-1.console.aws.amazon.com/ecs/home?region=eu-central-1#/taskDefinitions/reportservicejsreporttaskdefinitionF6C1EAF1/44) is the container on dev, and we have configuration `Log driver: awslogs` which means those are shipped to AWS Cloudwatch log group. We can see those [here](https://eu-central-1.console.aws.amazon.com/cloudwatch/home?region=eu-central-1#logsV2:log-groups/log-group/report-service-dev-jsreporttaskdefinitionjsreportLogGroup6B3C0A0D-SCVJ1tQacmbg) (for dev env).

These are NOT shipped to datadog, as of now, unfortunately, but that can be easily changed, just changing the log driver for the container.